keps/sig-node/20190129-hugepages.md

```diff
@@ -231,22 +227,53 @@ mount -t hugetlbfs \
 ```
 
 The proposal recommends extending the existing `EmptyDirVolumeSource` to satisfy
-this use case.  A new `medium=HugePages` option would be supported.  To write
-into this volume, the pod must make a request for huge pages. The `pagesize`
-argument is inferred from the `hugepages-<hugepagesize>` from the resource
-request.  If in the future, multiple huge page sizes are supported in a single
-pod spec, we may modify the `EmptyDirVolumeSource` to provide an optional page
-size.  The existing `sizeLimit` option for `emptyDir` would restrict usage to
-the minimum value specified between `sizeLimit` and the sum of huge page limits
-of all containers in a pod. This keeps the behavior consistent with memory
-backed `emptyDir` volumes whose usage is ultimately constrained by the pod
-cgroup sandbox memory settings.  The `min_size` option is omitted as its not
-necessary.  The `nr_inodes` mount option is omitted at this time in the same
+this use case.  A new `medium=HugePages[-<hugepagesize>]` options would be
+supported.  To write into this volume, the pod must make a request for huge
+pages.  The `pagesize` argument is inferred from the `hugepages-<hugepagesize>`
```

Comment:
The line `The pagesize argument is inferred from the `hugepages-<hugepagesize>`
request from the resource request.` should probably be updated to reflect that it is inferred from the `medium` of the mount, _or_ from the resource request as a fallback if there is only on page size in the podspec.

Reply:
Done